From 4719c69db1d3d3ea0cee22f16d582c07a614f947 Mon Sep 17 00:00:00 2001
From: Tiejun Chen <tiejun.chen@windriver.com>
Date: Tue, 3 Mar 2009 12:39:06 +0800
Subject: [PATCH] fsl_mpc8349e_mitx: Support RTC

powerpc/83xx: Add DS1339 RTC support for MPC8349E-mITX boards .dts

The RTC is sitting on the I2C2 bus at address 0x68. RTC interrupt signal
is connected to the IPIC's EXT2 interrupt line, the line is shared with
Vitesse 8201 Ethernet PHY.

[Tiejun: cherry picked from 672e806e to support MPC8349-mitx RTC]

Signed-off-by: Anton Vorontsov <avorontsov@ru.mvista.com>
Signed-off-by: Kumar Gala <galak@kernel.crashing.org>
Integrated-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 arch/powerpc/boot/dts/mpc8349emitx.dts |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/mpc8349emitx.dts b/arch/powerpc/boot/dts/mpc8349emitx.dts
index ceed03f..1ca9a9f 100644
--- a/arch/powerpc/boot/dts/mpc8349emitx.dts
+++ b/arch/powerpc/boot/dts/mpc8349emitx.dts
@@ -91,6 +91,13 @@
 			interrupts = <15 0x8>;
 			interrupt-parent = <&ipic>;
 			dfsrr;
+
+			rtc@68 {
+				compatible = "dallas,ds1339";
+				reg = <0x68>;
+				interrupts = <18 0x8>;
+				interrupt-parent = <&ipic>;
+			};
 		};
 
 		spi@7000 {
-- 
1.6.0.4

